src/server.ts

```diff
@@ -52,7 +52,11 @@ http
         }
         res.setHeader(
           'Content-Disposition',
-          `inline; filename=${talent.firstName}_${talent.lastName}_certificate.pdf`
+          `inline; filename=${talent.firstName
+            .normalize('NFD')
+            .replace(/\p{Diacritic}/gu, '')}_${talent.lastName
+            .normalize('NFD')
+            .replace(/\p{Diacritic}/gu, '')}_certificate.pdf`
```

Comment:
What do you think about a helper function like `normalizeDiacritic`?
This would reduce duplicate code.

```suggestion
          `inline; filename=${normalizeDiacritic(talent.firstName)}_${normalizeDiacritic(talent.lastName)}_certificate.pdf`
```

This functions could be placed in a `utils.ts`.

```ts
export function normalizeDiacritic(text: string): string {
  return text.normalize('NFD').replace(/\p{Diacritic}/gu, '');
}
```